lib/ansible/modules/source_control/github_webhook.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/python
+#
+# Copyright: (c) Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: github_webhook
+short_description: Manage Github webhooks
+version_added: "2.6"
+description:
+  - "Create and delete Github webhooks"
```

Comment:
Can you please add more description here?

Reply:
What more is needed? This seems sufficient to me.

Reply:
What more is needed? This seems sufficient to me.